### DISCUSSION TITLE
I haven't got my education benefits yet. And it has been 3 days.

### DISCUSSION THREAD

**Discussion by Rhys-Wang-wannaLearnMath** [community/community#144109] [GitHub Education] [answered]
### Hi everyone,

I've finish my education verification on 11.3, but I got no benefits yet, I wanna know the reasons. Thanks a lot.

**Comment by davevad93** [comment, marked as answer]
Hi @Rhys-Wang-wannaLearnMath ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Rhys-Wang-wannaLearnMath was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by Rhys-Wang-wannaLearnMath** [reply]
thanks a lot! Wish you have a nice day!

**Reply by davevad93** [reply]
You're welcome @Rhys-Wang-wannaLearnMath , have a nice day too! 👍
